packages/charts/react-charting/src/components/VerticalBarChart/__snapshots__/VerticalBarChartRTL.test.tsx.snap

```diff
@@ -14001,6 +14012,761 @@ exports[`Vertical bar chart rendering Should render the vertical bar chart with
 </div>
 `;
 
+exports[`Vertical bar chart rendering should render the vertical bar chart with some positive and some negative y value bars 1`] = `
```

Comment:
>exports [](http://example.com/codeflow?start=0&length=7)

I dont see the 'Second' series added in the snapshot

Reply:
will update all snapshots, mp that will resolve build issue too